index.js

```diff
@@ -103,7 +137,22 @@ function pickFields(item, fields) {
 
 function getUnique(termData) {
   const unique = new Set(termData.map(JSON.stringify));
-  return [...unique].map(JSON.parse);
+  return [...unique].map(strData => JSON.parse(strData));
+}
+
+function getData(key, filename) {
```

Comment:
Adding this so we can load multiple files like xref all data (`xref.json`), `xref-supported-types.json`, `xref-supported-specs.json` later